hooks/pre-command

```diff
@@ -26,19 +26,20 @@ upsert_ecr() {
     aws ecr create-repository --repository-name "${repository_name}"
   fi
 
-	policy_file="${BUILDKITE_PLUGIN_CREATE_ECR_POLICY:-''}"
-	echo "Policy file: ${policy_file}"
-  if [ "$policy_file" != "" ]; then
+	permissions_policy_file="${BUILDKITE_PLUGIN_CREATE_ECR_PERMISSIONS_POLICY:-''}"
```

Comment:
Hmm. Indentation is all over the place, perhaps I screwed it up. 😢 https://github.com/seek-oss/create-ecr-buildkite-plugin/blob/f616f3106647738799d55d535b8b86f0ec6e4496/hooks/pre-command#L29